lib/sidekiq_adhoc_job.rb

```diff
@@ -41,6 +37,10 @@ def initialize
     def configured?
```

Comment:
We should be able to remove this now that we don't check `module_names` anymore 😄

Reply:
Good catch! I've removed it. Feel free to have a final review? :D